conf/monitor/capture-statistics-template.yaml

```diff
@@ -48,8 +48,8 @@ objects:
             chmod 744 kubectl-top.sh ;
             echo "Collecting Statistics";
             OUTPUT_FILE=/var/log/containers/stress.log;
+            sleep 60;
             echo -e "\nOutput log file is: $OUTPUT_FILE\n";
-            touch $OUTPUT_FILE;
```

Comment:
Did you check with all collectors that there is no need to ` touch $OUTPUT_FILE;` ... I added that for some reason ... I actually don't remember why. What happens if this is kept ???